conjure/resourceconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "resources": {
+    "includes": [
+      {"pattern": "^(.(?!.*\\.class))*$"}
```

Comment:
Unclear if the `.class` extension exclusion is necessary, I was debugging other things.

Reply:
In an ideal world we'd be able to do this programatically using a build Feature like reflection to reduce dependence on custom configs.